internal/ast/utilities.go

```diff
@@ -2965,12 +2965,22 @@ func GetPropertyNameForPropertyNameNode(name *Node) string {
 	panic("Unhandled case in getPropertyNameForPropertyNameNode")
 }
 
-// !!!
 func HasQuestionToken(node *Node) bool {
 	switch node.Kind {
-	case KindParameter, KindMethodDeclaration, KindMethodSignature, KindShorthandPropertyAssignment,
-		KindPropertyAssignment, KindPropertyDeclaration, KindPropertySignature:
+	case KindParameter:
 		return node.AsParameterDeclaration().QuestionToken != nil
+	case KindMethodDeclaration:
+		return IsQuestionToken(node.AsMethodDeclaration().PostfixToken)
```

Comment:
I think `IsQuestionToken` needs to be modified to not crash when given a `nil` node.